.github/workflows/conda.yml

```
@@ -140,4 +140,64 @@ jobs:
         run: |
           # install anaconda for upload
           mamba install anaconda-client
-          anaconda upload dist/conda/noarch/*.tar.bz2
\ No newline at end of file
+          anaconda upload dist/conda/noarch/*.tar.bz2
+
+  report:
+    name: report
+    needs: conda
+    if: |
+      always()
+      && github.event_name == 'pull_request'
+      && github.repository == 'coiled/coiled-runtime'
+      && needs.conda.steps.test.outcome == 'failure'
```

xref https://github.com/dask/dask/blob/bfc76afdcdc43c575a10ffefda94aaba424fe347/.github/workflows/upstream.yml#L75

```suggestion
      && needs.conda.result == 'failure'
```